tests/tests_deployment/keycloak_api_utils.py

```diff
@@ -0,0 +1,944 @@
+"""Helper class for Keycloak API interactions."""
+
+import base64
+import json
+
+import requests
+
+TIMEOUT = 10
+
+
+def decode_jwt_token(token: str) -> dict:
+    """Decode a JWT token without verification (for testing purposes).
```

Comment:
Can we add a stronger warning here. "DO NOT USE THIS FOR PRODUCTION" or something like that. not verifying sigs in test is fine, I just don't want someone copying this pattern later in prod